tests/contrib/langchain/conftest.py

```diff
@@ -119,6 +119,14 @@ def langchain_community(ddtrace_global_config, ddtrace_config_langchain, mock_lo
     yield langchain_community
 
 
+@pytest.fixture
+def langchain_core(ddtrace_global_config, ddtrace_config_langchain, mock_logs, mock_metrics, langchain):
+    import langchain_core
+    import langchain_core.prompts  # noqa: F401
```

Comment:
Just curious, what was the purpose of this import?

Reply:
That makes sense, figured we ran into the same issue as before (langchain not automatically importing submodules for some reason). Thanks for the explanation!